src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -198,8 +207,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
-            if (commandResult.isTagList()) { // TODO: Remove this in v1.3
+            if (prevIsTagList) {
                 showTagList();
+            } else {
+                showPersonList();
             }
```

Comment:
This is not required. The issue of tag list not auto-refreshing will resolve itself after the change in Model